### PR TITLE
feat(cli): remove hardcoded seed

### PR DESCRIPTION
**Pull request by simonovic86**
- remove hardcoded identity wallet seed from Ceramic CLI
- generate seed if there is no and save it to `~/.ceramic/config.json`

### CODE REVIEW

File: packages/ceramic-cli/src/ceramic-daemon.ts [hunk 1]
```diff
@@ -87,11 +100,41 @@ class CeramicDaemon {
...
+
+    await fs.mkdir(DEFAULT_CLI_CONFIG_PATH, { recursive: true })
+
+    console.log('Generating identity wallet seed...')
+    const config: CliConfig = {
+      seed: '0x' + Buffer.from(nacl.randomBytes(32)).toString('hex') // create new seed
+    }
+
+    await fs.writeFile(fullCliConfigPath, JSON.stringify(config, null, 2))
+    console.log('Identity wallet seed generated')
```
Comment (oed) [inline packages/ceramic-cli/src/ceramic-daemon.ts:134]: One console log is probably enough here? Let's keep the second one?

File: packages/ceramic-cli/package.json [hunk 2]
```diff
@@ -38,7 +38,8 @@
     "commander": "^5.0.0",
     "express": "^4.17.1",
     "identity-wallet": "1.2.0-ceramic.1",
-    "ipfs-http-client": "^44.0.0"
+    "ipfs-http-client": "^44.0.0",
+    "tweetnacl": "^1.0.3"
```
Comment (oed) [inline packages/ceramic-cli/package.json:42]: Doesn't nodejs have a built in way to generate randomness? Seems exessive to import tweetnacl just for that.

File: packages/ceramic-cli/src/ceramic-daemon.ts (hunk 1 above)
Comment (simonovic86) [reply packages/ceramic-cli/src/ceramic-daemon.ts:134]: makes sense! 👍

File: packages/ceramic-cli/package.json (hunk 2 above)
Comment (simonovic86) [reply packages/ceramic-cli/package.json:42]: yep, but I thought there's a reason why the identity wallet uses `tweetnacl` for randomness. I'll change it! 👍

Comment (oed) [reply packages/ceramic-cli/package.json:42]: I think IDW uses it because it can be run in browsers as well. IDW also uses tweetnacl for other things :)

Comment (simonovic86) [reply packages/ceramic-cli/package.json:42]: makes sense 👍

File: packages/ceramic-cli/src/ceramic-daemon.ts [hunk 3]
```diff
@@ -87,11 +100,41 @@ class CeramicDaemon {
...
+    if (exists) {
+      const configJson = await fs.readFile(fullCliConfigPath, { encoding: 'utf8' })
+      return JSON.parse(configJson)
+    }
+
+    await fs.mkdir(DEFAULT_CLI_CONFIG_PATH, { recursive: true })
+
+    console.log('Generating identity wallet seed...')
+    const config: CliConfig = {
+      seed: '0x' + Buffer.from(crypto.randomBytes(32)).toString('hex') // create new seed
```
Comment (oed) [inline packages/ceramic-cli/src/ceramic-daemon.ts:130]: Nice :+1:

**Pull request merged**